app/Observers/ArticleObserver.php

```diff
@@ -0,0 +1,42 @@
+<?php
```

Comment:
Decided to keep observer related stuff in a separate class opposed to the msq implementation for the sake of simplicity.